### PR TITLE
plt func

### PR DESCRIPTION
**Pull request by MimiYuchenGuo**
#466

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -36,6 +36,7 @@ export(formatCharacteristics)
 export(gtCharacteristics)
 export(gtResult)
 export(mockPatientProfiles)
+export(plotSummariseLargeScaleCharacteristics)
```
Comment (catalamarti) [inline NAMESPACE:39]: plotLargeScaleCharacteristics

File: R/plotSummariseLargeScaleCharacteristics.R [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+#' @param cohort_name the cohort name you want to characterise
+#' @param compare_cohorts Whether to compare cohort or not
+#' @param cohort_2_name if is not NULL, need to provide cohort_2_name as second cohort to compare to
+#' @param facet_vars column in data to facet by
+#' @param color_vars column in data to color by
+#' @param color_names A vector or pre-selected color
+#' @param options Other plot options in a list
+#' @return A ggplot
+#' @export
+plotSummariseLargeScaleCharacteristics <- function(data, cohort_name, compare_cohorts = FALSE,
```
Comment (catalamarti) [inline R/plotSummariseLargeScaleCharacteristics.R:29]: variables should be camelCase

File: R/plotSummariseLargeScaleCharacteristics.R [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                                                   cohort_2_name = NULL,
```
Comment (catalamarti) [inline R/plotSummariseLargeScaleCharacteristics.R:30]: rename to reference?

File: R/plotSummariseLargeScaleCharacteristics.R [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+                                                   facet_vars = c("estimate_type"),
+                                                   color_vars = c("variable_level", "group_level"),
+                                                   color_names = NULL, options = list()) {
+  if (nrow(data) == 0) {
+    stop("Input data is empty. Please provide a non-empty data frame.")
```
Comment (catalamarti) [inline R/plotSummariseLargeScaleCharacteristics.R:35]: could we just return an empty plot?

File: R/plotSummariseLargeScaleCharacteristics.R [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#' Summarise characteristics of individuals
+#'
+#' @param data output of summariseLargeScaleCharacteristics
+#' @param cohort_name the cohort name you want to characterise
```
Comment (catalamarti) [inline R/plotSummariseLargeScaleCharacteristics.R:20]: they can do the filtering for you right? I would delete this option

**Review by catalamarti — CHANGES_REQUESTED** [review]

**Comment by catalamarti** [comment]
Could you please add a vignette showing the plot and how to use it? @MimiYuchenGuo

**Comment by MimiYuchenGuo** [comment]
Now you can do boxplot barplot and scatterplot

**Review by catalamarti — APPROVED** [review]

**Pull request merged**